src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,210 @@
+    private static final int ERROR_TODO_IS_EMPTY = 1;
+    private static final int ERROR_EVENT_IS_EMPTY = 2;
+    private static final int ERROR_DEADLINE_IS_EMPTY = 3;
+    private static final int ERROR_COMMAND_NOT_FOUND = 4;
+    private static final int ERROR_IS_INVALID = 5;
+    private static final int CMD_NOT_FOUND = 0;
+    private static final int CMD_TODO = 1;
+    private static final int CMD_EVENT = 2;
+    private static final int CMD_DEADLINE = 3;
+    private static final int CMD_LIST = 4;
+    private static final int CMD_DONE = 5;
+    private static final int CMD_TERMINATE = 6;
+    private static final int CMD_DELETE = 7;
+    private static final String LIST = "list";
+    private static final String TODO = "todo";
+    private static final String EVENT = "event";
+    private static final String DEADLINE = "deadline";
+    private static final String DONE = "done";
+    private static final String BYE = "bye";
+    private static final String DELETE = "delete";
+    private static final String BY = "/by";
+    private static final String AT = "/at";
+    private static final ArrayList<Task> items = new ArrayList<>();
+    private static final String fileDir = "./data";
+    private static final String fileName = "./data/save.txt";
+    private static final String logo =
+              " _____  ___  _____     ______\n"
+            + "|___  | | | |_____|  / / -- \\ \\ \n"
+            + "   / /  | |    / /  | |      | | \n"
+            + "  / /   | |   / /   | |      | |\n"
+            + " / /___ | |  /_/__  | |  --  | |\n"
+            + "|_____| | | |_____|  \\ \\____/ /\n";
+    private static final String border = "____________________________________________________________\n";
+
+
+    public static void printStartMessage() {
+        System.out.println(logo);
+        System.out.println(border + "Hi bro, my name is Echo");
+        System.out.println("What do you want?\n" + border);
+        System.out.println("Type bye to exit\n" + border);
+    }
+    public static boolean isInvalid(CommandList task, String line , String key) {
+        if (!line.split(key)[1].trim().isEmpty()) {
+            if (Integer.parseInt(line.split(key)[1].trim()) > task.getTaskCount()) {
+                return true;
+            }
+        }
+        return (line.length() <= FIVE);
+    }
+    private static void createSave() {
+        File saveFile = new File(Duke.fileName);
+        File saveDir = new File(Duke.fileDir);
+        if (saveDir.mkdirs()) {
+            System.out.println("Successfully created save dir");
+        } else {
+            System.out.println("Save folder already exists.");
+        }
+        try {
+            if (saveFile.createNewFile()) {
+                System.out.println("Successfully created save file");
+            } else {
+                System.out.println("Save file already exists");
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
How about including the previous statements in the [ try catch block](https://se-education.org/guides/conventions/java/basic.html#layout)? After all, this method is doing file related operations and the program could `catch` file related exceptions.